util/test/account_testutil.go

```diff
@@ -596,3 +598,21 @@ func MakeGenesisBlock() bookkeeping.Block {
 		},
 	}
 }
+
+// CreateInitState makes an initState
+func CreateInitState(genesis *bookkeeping.Genesis, genesisBlock *bookkeeping.Block) (ledgercore.InitState, error) {
```

Comment:
I can see why you wanted to put it in `util`. Perhaps it would be better to avoid two copies. But I'm not very particular.

Reply:
I'm moving it to util.